src/algebraic_geometry/locally_ringed_space/has_colimits.lean

```diff
@@ -0,0 +1,113 @@
+/-
+Copyright (c) 2021 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import algebraic_geometry.locally_ringed_space
+import algebra.category.CommRing.constructions
+import algebraic_geometry.open_immersion
+import category_theory.limits.constructions.limits_of_products_and_equalizers
+
+/-!
+# Colimits of LocallyRingedSpace
+
+We construct the explict coproducts and coequalizers in the category of `LocallyRingedSpace`.
+It then follows that `LocallyRingedSpace` has all colimits, and
+`forget_to_SheafedSpace` preserves them.
```

Comment:
Am I missing something? I don't see anything about coequalizers of `LocallyRingedSpaces` in this file.